src/mono/nuget/Microsoft.NET.Runtime.MonoTargets.Sdk/Sdk/component-manifest.targets

```diff
@@ -0,0 +1,62 @@
+<Project>
+  <Target Name="_MonoComputeAvailableComponentDefinitions">
+    <!-- Input: _MonoRuntimeAvailableComponents item list: the components provided by the runtime -->
+    <!-- Input: _MonoRuntimeComponentLinking property: either 'dynamic' or 'static'.  Are the components static or shared libraries. -->
```

Comment:
What is intended to set `$(_MonoRuntimeComponentCurrentLinking)`? Is the Android workload going to default this to `dynamic` and iOS might have some other way to select it?

Reply:
> What is intended to set `$(_MonoRuntimeComponentCurrentLinking)`? Is the Android workload going to default this to `dynamic` and iOS might have some other way to select it?

Actually the runtime pack gets to choose, not the workload.

I should fixup this part of the doc - `@(_MonoRuntimeAvailableComponents)` and `@(_MonoRuntimeComponentLinking)` aren't really inputs to the target that the workload authors are expected to set.